Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -113,6 +113,27 @@ public bool AllowFarManipulation
             set => allowFarManipulation = value;
         }
 
+        [SerializeField]
+        [Tooltip(
+             "Whether physics forces are used to move the object when performing near manipulations. " +
+             "Off will make the object feel more directly connected to the hand. On will honor the mass and inertia of the object. " +
+             "The default is off.")]
+        private bool useForcesForNearManipulation = false;
+
+        /// <summary>
+        /// Whether physics forces are used to move the object when performing near manipulations.
+        /// </summary>
+        /// <remarks>
+        /// Setting this to <c>false</c> will make the object feel more directly connected to the
+        /// users hand. Setting this to <c>true</c> will honor the mass and inertia of the object,
+        /// but may feel as though the object is connected through a spring. The default is <c>false</c>.
+        /// </remarks>
+        public bool UseForcesForNearManipulation
+        {
+            get => useForcesForNearManipulation;
```

Comment:
i'm wondering if we should change to have this as a enum flag instead of the separate bools - we already have the enum for enabling interaction and it could be reused to set the forces behavior and the smoothing - would be cleaner than keeping all those bools around. What do you think?

Reply:
jared pointed out this will be a breaking change - so let's not do this for now :)